sdk/src/platform/client.ts

```
@@ -1810,6 +1811,57 @@ export class PlatformApiClient {
+   */
+  listEvalSuiteStageAnalytics(
+    params: {
+      projectId: string;
+      suiteId: string;
+      from?: number;
+      to?: number;
+      runGroupId?: string;
+      cursor?: string;
+      limit?: number;
+    },
+    options?: RequestOptions,
```

ESLint

[error] Delete `,`

(prettier/prettier)